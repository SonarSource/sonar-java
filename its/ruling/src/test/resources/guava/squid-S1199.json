{
'com.google.guava:guava:src/com/google/common/collect/ImmutableEnumMap.java':[
41,
],
'com.google.guava:guava:src/com/google/common/math/DoubleMath.java':[
93,
102,
],
}
