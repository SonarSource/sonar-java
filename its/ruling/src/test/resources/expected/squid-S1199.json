{
'project:com/sun/corba/se/PortableActivationIDL/_ActivatorImplBase.java':[
52,
71,
85,
112,
127,
141,
163,
183,
207,
229,
241,
260,
],
'project:com/sun/corba/se/PortableActivationIDL/_InitialNameServiceImplBase.java':[
45,
],
'project:com/sun/corba/se/PortableActivationIDL/_LocatorImplBase.java':[
46,
73,
98,
117,
],
'project:com/sun/corba/se/PortableActivationIDL/_ORBProxyImplBase.java':[
48,
],
'project:com/sun/corba/se/PortableActivationIDL/_RepositoryImplBase.java':[
51,
72,
88,
106,
125,
145,
164,
177,
189,
],
'project:com/sun/corba/se/PortableActivationIDL/_ServerManagerImplBase.java':[
61,
80,
94,
121,
136,
150,
172,
192,
216,
238,
250,
269,
286,
313,
338,
357,
],
'project:com/sun/corba/se/PortableActivationIDL/_ServerProxyImplBase.java':[
46,
57,
68,
],
'project:com/sun/corba/se/impl/corba/AnyImpl.java':[
420,
429,
448,
458,
],
'project:com/sun/corba/se/impl/corba/TypeCodeImpl.java':[
327,
336,
342,
745,
768,
780,
796,
822,
835,
849,
877,
1384,
1392,
1401,
1485,
1505,
1516,
1528,
1543,
1570,
1712,
1720,
1730,
1806,
1823,
1833,
1844,
1858,
1879,
1982,
1994,
2006,
2013,
2025,
2031,
2037,
2050,
2077,
2082,
2089,
2096,
2103,
2110,
2117,
2124,
2131,
2138,
2146,
2153,
2168,
],
'project:com/sun/corba/se/impl/naming/cosnaming/NamingContextImpl.java':[
640,
649,
],
'project:com/sun/corba/se/impl/naming/pcosnaming/NamingContextImpl.java':[
604,
613,
],
'project:com/sun/corba/se/spi/activation/_ActivatorImplBase.java':[
47,
63,
86,
97,
118,
136,
157,
174,
],
'project:com/sun/corba/se/spi/activation/_InitialNameServiceImplBase.java':[
40,
],
'project:com/sun/corba/se/spi/activation/_LocatorImplBase.java':[
43,
67,
91,
108,
],
'project:com/sun/corba/se/spi/activation/_RepositoryImplBase.java':[
48,
68,
83,
100,
117,
135,
153,
164,
175,
],
'project:com/sun/corba/se/spi/activation/_ServerImplBase.java':[
46,
57,
68,
],
'project:com/sun/corba/se/spi/activation/_ServerManagerImplBase.java':[
51,
67,
90,
101,
122,
140,
161,
178,
199,
223,
247,
264,
],
'project:com/sun/imageio/plugins/jpeg/JPEGImageReader.java':[
885,
897,
],
'project:com/sun/imageio/plugins/png/RowFilter.java':[
106,
117,
127,
138,
155,
],
'project:com/sun/jmx/snmp/IPAcl/Parser.java':[
70,
79,
82,
84,
123,
126,
128,
155,
158,
160,
192,
201,
204,
206,
256,
295,
298,
300,
349,
352,
354,
550,
553,
555,
593,
596,
598,
617,
656,
659,
661,
698,
701,
703,
747,
750,
752,
815,
818,
820,
846,
849,
851,
870,
909,
912,
914,
951,
954,
956,
],
'project:com/sun/jmx/snmp/SnmpPduFactoryBER.java':[
117,
122,
],
'project:com/sun/org/apache/bcel/internal/classfile/Utility.java':[
253,
398,
843,
857,
],
'project:com/sun/org/apache/bcel/internal/generic/ConstantPoolGen.java':[
719,
726,
733,
757,
],
'project:com/sun/org/apache/regexp/internal/RE.java':[
824,
913,
999,
1081,
1113,
1200,
1218,
1254,
1288,
],
'project:com/sun/org/apache/regexp/internal/RECompiler.java':[
425,
595,
812,
891,
904,
999,
1119,
],
'project:com/sun/org/apache/xalan/internal/xsltc/compiler/XPathLexer.java':[
804,
808,
812,
816,
820,
824,
828,
832,
836,
840,
844,
848,
852,
856,
860,
864,
868,
872,
876,
880,
884,
888,
892,
896,
900,
904,
908,
912,
916,
920,
924,
929,
934,
938,
942,
946,
950,
954,
958,
962,
966,
970,
974,
978,
982,
986,
990,
994,
998,
1002,
1006,
1010,
1014,
1018,
1022,
1026,
1030,
1034,
1038,
1042,
1046,
1050,
1054,
1058,
1062,
1066,
1070,
1074,
1078,
1082,
1086,
1090,
1094,
1098,
1102,
1106,
1110,
1114,
1118,
1122,
1126,
1130,
1134,
1138,
1142,
1146,
1150,
1154,
1158,
1162,
1166,
1170,
1174,
1178,
1182,
1186,
1190,
1194,
1198,
1202,
1206,
1210,
1214,
1218,
1222,
1226,
1230,
1234,
1238,
1242,
1246,
1250,
1254,
1258,
1262,
1266,
1270,
1274,
1278,
1282,
1286,
1290,
1294,
1298,
1302,
1306,
1310,
1314,
1318,
1322,
1326,
1330,
1334,
1338,
1342,
1346,
1350,
1354,
1358,
1362,
1366,
1370,
1374,
1378,
1382,
1386,
1390,
1394,
1398,
1402,
1406,
1410,
1414,
1418,
1422,
1426,
1430,
1434,
1438,
1442,
1446,
1450,
1454,
1458,
1462,
1466,
1470,
1474,
1478,
1482,
1486,
1490,
1494,
1498,
1502,
1506,
1510,
1514,
1518,
1522,
1526,
1530,
1534,
1538,
1542,
1546,
1550,
1554,
1558,
1562,
1566,
1570,
1574,
1578,
1582,
1586,
1590,
1594,
1598,
],
'project:com/sun/org/apache/xalan/internal/xsltc/compiler/XPathParser.java':[
1136,
1145,
1154,
1163,
1172,
1181,
1190,
1199,
1208,
1217,
1226,
1235,
1244,
1253,
1262,
1271,
1280,
1289,
1301,
1313,
1322,
1331,
1351,
1360,
1369,
1378,
1390,
1402,
1416,
1430,
1445,
1461,
1559,
1615,
1647,
1659,
1671,
1698,
1726,
1738,
1750,
1765,
1777,
1786,
1795,
1804,
1813,
1822,
1831,
1840,
1849,
1858,
1867,
1876,
1885,
1894,
1903,
1912,
1921,
1933,
1945,
1963,
1984,
2009,
2030,
2056,
2123,
2135,
2147,
2156,
2168,
2194,
2206,
2218,
2230,
2263,
2278,
2290,
2302,
2317,
2329,
2341,
2353,
2368,
2383,
2398,
2410,
2425,
2440,
2452,
2467,
2482,
2497,
2512,
2524,
2539,
2554,
2566,
2581,
2593,
2608,
2620,
2632,
2646,
2661,
2677,
2686,
2695,
2704,
2716,
2725,
2734,
2743,
2752,
2761,
2773,
2794,
2811,
2831,
2848,
2863,
2875,
2892,
2906,
2921,
2936,
2948,
2960,
2975,
2989,
3001,
3013,
3028,
3043,
3055,
3067,
3076,
3091,
3103,
3115,
3127,
],
'project:com/sun/org/apache/xalan/internal/xsltc/dom/SAXImpl.java':[
1287,
1292,
1297,
],
'project:com/sun/org/apache/xalan/internal/xsltc/trax/TransformerImpl.java':[
1125,
],
'project:com/sun/org/apache/xerces/internal/dom/CoreDocumentImpl.java':[
981,
1060,
1125,
1538,
1590,
1632,
1637,
1642,
1650,
1664,
1670,
1675,
1714,
1720,
1732,
1802,
1822,
1830,
1834,
1866,
1881,
],
'project:com/sun/org/apache/xerces/internal/dom/DOMNormalizer.java':[
278,
295,
448,
477,
508,
584,
654,
],
'project:com/sun/org/apache/xerces/internal/dom/DeferredDocumentImpl.java':[
970,
979,
984,
996,
1002,
1009,
1060,
1065,
1070,
1075,
1080,
1089,
1094,
1428,
1429,
1430,
1431,
1432,
1433,
1434,
1435,
1436,
1437,
1438,
1439,
1481,
1482,
1483,
1484,
1485,
1486,
1487,
1488,
1489,
1490,
1491,
1492,
],
'project:com/sun/org/apache/xerces/internal/dom/DeferredDocumentTypeImpl.java':[
141,
147,
153,
159,
168,
],
'project:com/sun/org/apache/xerces/internal/dom/NodeImpl.java':[
1060,
1066,
1094,
1102,
1132,
1139,
1147,
1398,
1427,
1437,
1444,
1477,
1482,
1492,
1499,
1521,
1567,
1576,
1583,
],
'project:com/sun/org/apache/xerces/internal/impl/PropertyManager.java':[
62,
66,
],
'project:com/sun/org/apache/xerces/internal/impl/XMLDocumentFragmentScannerImpl.java':[
2535,
2540,
2556,
2561,
2603,
2608,
2646,
2662,
2711,
2726,
2889,
2912,
2918,
2929,
2965,
3022,
3062,
3071,
],
'project:com/sun/org/apache/xerces/internal/impl/XMLDocumentScannerImpl.java':[
838,
850,
903,
911,
918,
995,
999,
1093,
1135,
1156,
1167,
1172,
1355,
1370,
1396,
1402,
1410,
1424,
1430,
1437,
],
'project:com/sun/org/apache/xerces/internal/impl/XMLEntityManager.java':[
2700,
2704,
2708,
2712,
2716,
],
'project:com/sun/org/apache/xerces/internal/impl/XMLEntityScanner.java':[
2024,
2028,
2032,
2036,
2040,
],
'project:com/sun/org/apache/xerces/internal/impl/XMLErrorReporter.java':[
432,
436,
440,
],
'project:com/sun/org/apache/xerces/internal/impl/XMLScanner.java':[
409,
453,
480,
496,
],
'project:com/sun/org/apache/xerces/internal/impl/dtd/DTDGrammar.java':[
1534,
1545,
1567,
1601,
1635,
1640,
1648,
1653,
1657,
2089,
2104,
2120,
2136,
2154,
2175,
2183,
2188,
2192,
],
'project:com/sun/org/apache/xerces/internal/impl/dtd/XMLDTDValidator.java':[
1349,
1372,
1400,
1413,
1442,
1752,
1755,
1767,
1770,
1773,
1776,
],
'project:com/sun/org/apache/xerces/internal/impl/xpath/XPath.java':[
202,
209,
220,
230,
262,
268,
273,
287,
292,
635,
644,
653,
656,
],
'project:com/sun/org/apache/xerces/internal/impl/xpath/regex/RegexParser.java':[
792,
],
'project:com/sun/org/apache/xerces/internal/impl/xpath/regex/RegularExpression.java':[
1068,
1080,
1109,
1131,
1141,
1171,
1193,
1206,
1215,
1235,
1258,
1268,
1276,
1289,
1347,
1383,
1395,
1418,
1498,
],
'project:com/sun/org/apache/xerces/internal/impl/xs/SchemaGrammar.java':[
564,
614,
],
'project:com/sun/org/apache/xerces/internal/impl/xs/XSConstraints.java':[
760,
765,
773,
782,
794,
806,
814,
819,
829,
835,
843,
848,
864,
873,
879,
887,
892,
907,
916,
922,
931,
936,
951,
958,
965,
975,
981,
],
'project:com/sun/org/apache/xerces/internal/impl/xs/identity/XPathMatcher.java':[
461,
465,
],
'project:com/sun/org/apache/xerces/internal/impl/xs/models/XSDFACM.java':[
969,
989,
1003,
1031,
],
'project:com/sun/org/apache/xerces/internal/parsers/AbstractDOMParser.java':[
627,
630,
636,
645,
709,
712,
715,
723,
1038,
1042,
1048,
1059,
1334,
1337,
1343,
1365,
1439,
1442,
1445,
1452,
1560,
1563,
1570,
1577,
2632,
2635,
2638,
2642,
],
'project:com/sun/org/apache/xerces/internal/util/DOMUtil.java':[
85,
89,
93,
109,
113,
118,
122,
],
'project:com/sun/org/apache/xerces/internal/xpointer/ElementSchemePointer.java':[
170,
183,
],
'project:com/sun/org/apache/xerces/internal/xpointer/XPointerHandler.java':[
166,
183,
],
'project:com/sun/org/apache/xml/internal/dtm/ref/ChunkedIntArray.java':[
87,
121,
203,
256,
],
'project:com/sun/org/apache/xml/internal/dtm/ref/DTMDefaultBaseIterators.java':[
120,
],
'project:com/sun/org/apache/xml/internal/dtm/ref/DTMNodeProxy.java':[
1711,
1770,
1777,
1902,
1909,
],
'project:com/sun/org/apache/xml/internal/dtm/ref/DTMTreeWalker.java':[
229,
287,
303,
315,
324,
340,
393,
],
'project:com/sun/org/apache/xml/internal/dtm/ref/dom2dtm/DOM2DTM.java':[
925,
982,
1001,
1060,
1103,
1115,
1164,
1667,
],
'project:com/sun/org/apache/xml/internal/dtm/ref/dom2dtm/DOM2DTMdefaultNamespaceDeclarationNode.java':[
314,
373,
380,
504,
511,
],
'project:com/sun/org/apache/xml/internal/security/Init.java':[
133,
],
'project:com/sun/org/apache/xml/internal/security/algorithms/SignatureAlgorithm.java':[
394,
],
'project:com/sun/org/apache/xml/internal/security/signature/Manifest.java':[
107,
],
'project:com/sun/org/apache/xml/internal/security/signature/Reference.java':[
188,
197,
562,
],
'project:com/sun/org/apache/xml/internal/security/signature/XMLSignatureInputDebugger.java':[
182,
],
'project:com/sun/org/apache/xml/internal/security/transforms/implementations/FuncHere.java':[
94,
111,
116,
124,
132,
],
'project:com/sun/org/apache/xml/internal/security/transforms/implementations/TransformXSLT.java':[
145,
],
'project:com/sun/org/apache/xml/internal/security/utils/RFC2253Parser.java':[
511,
533,
],
'project:com/sun/org/apache/xml/internal/security/utils/resolver/implementations/ResolverDirectHTTP.java':[
155,
175,
],
'project:com/sun/org/apache/xml/internal/serialize/BaseMarkupSerializer.java':[
1023,
1033,
1036,
1049,
1061,
1066,
1081,
1093,
1097,
1108,
1120,
1123,
1132,
1154,
1161,
1164,
1171,
1177,
1180,
1189,
1196,
1260,
],
'project:com/sun/org/apache/xml/internal/serialize/DOMSerializerImpl.java':[
1083,
1086,
1089,
1138,
1144,
1152,
1158,
1162,
],
'project:com/sun/org/apache/xml/internal/serialize/TextSerializer.java':[
311,
320,
345,
],
'project:com/sun/org/apache/xml/internal/serialize/XMLSerializer.java':[
1144,
1147,
],
'project:com/sun/org/apache/xml/internal/serializer/ToHTMLStream.java':[
2141,
2146,
2178,
2266,
2271,
2282,
],
'project:com/sun/org/apache/xml/internal/serializer/ToStream.java':[
487,
1465,
],
'project:com/sun/org/apache/xml/internal/serializer/TreeWalker.java':[
290,
310,
367,
384,
396,
405,
423,
500,
514,
],
'project:com/sun/org/apache/xml/internal/serializer/WriterToUTF8Buffered.java':[
259,
387,
],
'project:com/sun/org/apache/xml/internal/utils/DOMHelper.java':[
1313,
],
'project:com/sun/org/apache/xml/internal/utils/TreeWalker.java':[
301,
357,
374,
386,
395,
413,
480,
],
'project:com/sun/org/apache/xml/internal/utils/Trie.java':[
126,
131,
163,
],
'project:com/sun/org/apache/xml/internal/utils/UnImplNode.java':[
1450,
1509,
1516,
1641,
1648,
],
'project:com/sun/org/apache/xpath/internal/compiler/Lexer.java':[
140,
],
'project:com/sun/org/apache/xpath/internal/objects/XObject.java':[
710,
],
'project:com/sun/org/apache/xpath/internal/patterns/NodeTest.java':[
570,
577,
643,
650,
],
'project:java/awt/DefaultKeyboardFocusManager.java':[
279,
404,
442,
563,
614,
638,
870,
],
'project:java/awt/GridBagLayout.java':[
1874,
],
'project:java/awt/KeyboardFocusManager.java':[
2881,
2885,
],
'project:java/awt/TextComponent.java':[
1076,
1083,
1145,
1159,
1198,
1212,
],
'project:java/awt/datatransfer/SystemFlavorMap.java':[
474,
479,
484,
488,
],
'project:java/awt/event/HierarchyEvent.java':[
303,
],
'project:java/awt/font/TextLayout.java':[
616,
1144,
1158,
],
'project:java/awt/font/TextLine.java':[
423,
437,
451,
463,
],
'project:java/awt/font/TextMeasurer.java':[
223,
399,
],
'project:java/awt/image/BufferedImage.java':[
322,
335,
344,
363,
375,
390,
405,
420,
432,
444,
454,
483,
495,
],
'project:java/awt/image/ComponentColorModel.java':[
517,
534,
551,
568,
627,
633,
639,
802,
817,
831,
1201,
1300,
1381,
1615,
1627,
1967,
1981,
1996,
2057,
2062,
2067,
2468,
2495,
2523,
2550,
2577,
2602,
2635,
2656,
2677,
2698,
2719,
2738,
],
'project:java/awt/image/DirectColorModel.java':[
878,
888,
1200,
1223,
1246,
1277,
1295,
1313,
],
'project:java/io/StreamTokenizer.java':[
811,
],
'project:java/lang/CharacterData00.java':[
1449,
],
'project:java/lang/CharacterData01.java':[
643,
],
'project:java/lang/CharacterData02.java':[
380,
],
'project:java/lang/CharacterData0E.java':[
381,
],
'project:java/lang/CharacterDataLatin1.java':[
319,
],
'project:java/lang/Class.java':[
2561,
],
'project:java/lang/reflect/Proxy.java':[
504,
],
'project:java/math/BigDecimal.java':[
1145,
],
'project:java/text/SimpleDateFormat.java':[
1922,
1977,
2039,
],
'project:java/util/Formatter.java':[
3838,
3849,
3855,
3861,
3867,
3873,
3884,
3890,
3896,
3910,
3920,
3932,
3944,
3963,
3971,
3977,
3986,
3996,
4007,
4017,
4024,
],
'project:java/util/GregorianCalendar.java':[
1157,
1203,
1236,
1296,
1352,
1372,
1389,
1424,
1513,
1590,
1689,
1707,
1727,
1757,
1791,
1830,
1878,
],
'project:java/util/JapaneseImperialCalendar.java':[
540,
592,
703,
775,
822,
847,
865,
901,
1102,
1155,
1193,
1227,
1241,
1303,
1335,
1377,
1436,
1469,
1487,
],
'project:javax/imageio/ImageTypeSpecifier.java':[
1216,
],
'project:javax/swing/border/TitledBorder.java':[
361,
373,
],
'project:javax/swing/colorchooser/DiagramComponent.java':[
73,
],
'project:javax/swing/plaf/basic/BasicTabbedPaneUI.java':[
2023,
2038,
2052,
2067,
],
'project:javax/swing/text/View.java':[
509,
],
'project:javax/swing/text/html/CSSParser.java':[
239,
],
'project:javax/swing/text/html/StyleSheet.java':[
1976,
1984,
],
'project:javax/swing/text/html/parser/ContentModel.java':[
170,
],
'project:javax/swing/text/html/parser/ContentModelState.java':[
113,
126,
158,
218,
233,
],
'project:javax/swing/text/html/parser/DTD.java':[
410,
416,
],
'project:javax/swing/text/rtf/RTFReader.java':[
1120,
1188,
],
'project:org/omg/CORBA/ServiceDetailHelper.java':[
39,
47,
],
'project:org/omg/CORBA/ServiceInformationHelper.java':[
49,
54,
],
'project:org/omg/CosNaming/BindingIteratorPOA.java':[
53,
72,
88,
],
'project:org/omg/CosNaming/NamingContextExtPOA.java':[
73,
99,
129,
163,
208,
252,
294,
333,
372,
407,
447,
465,
495,
526,
],
'project:org/omg/CosNaming/NamingContextPOA.java':[
78,
122,
164,
203,
242,
277,
317,
335,
365,
396,
],
'project:org/omg/CosNaming/_BindingIteratorImplBase.java':[
56,
73,
95,
],
'project:org/omg/CosNaming/_NamingContextImplBase.java':[
63,
109,
155,
195,
235,
271,
306,
333,
344,
386,
],
}
